emails/2023-06-09.md

```diff
@@ -0,0 +1,25 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+We have a new [DoltHub homepage](https://www.dolthub.com/) which emphasizes the easiest way we've found to describe Dolt, "It's like Git and MySQL had a baby". Some people take this too literally and let their imagination run wil with it, with [humorous consequences](https://www.youtube.com/watch?v=jb2AvF8XzII&t=128s).
+
+## DoltLab 1.0
+
+Continuing our [Dolt 1.0](https://www.dolthub.com/blog/2023-05-05-dolt-1-dot-0/) extravaganza, we bring to you [DoltLab 1.0](https://www.dolthub.com/blog/2023-06-05-announcing-doltlab-1-dot-0/), now backed by Dolt. A few weeks ago, we [announced DoltHub was backed by Dolt](https://www.dolthub.com/blog/2023-05-12-dolthub-on-hosted-dolt/) signalling Dolt was ready to back our production OLTP application so it's ready to back yours. Now, if you're running DoltLab 1.0, you're running an OLTP Dolt as well. 
+
+[![DoltLab 1.0](../images/doltlab-1_0.png)](https://www.dolthub.com/blog/2023-06-05-announcing-doltlab-1-dot-0/)
+
+DoltLab is how you get DoltHub functionality in your local network. DoltLab is free! [Try it today](https://docs.dolthub.com/products/doltlab/installation).
+
+### Collation Support in Regex
+
+A couple weeks ago we announced regex collation support. Unexpectedly, getting there [involved using WASM](https://www.dolthub.com/blog/2023-05-19-wasm-in-go/). This also made us deep dive into MySQL's collation regex support and we discovered some [interesting edge cases](https://www.dolthub.com/blog/2023-06-02-replicating-unexpected-behavior/). If you want togeek out on esoteric MySQL behavior, this is the blog for you.
+
+I'm telling you about this because this blog signals how seriously we take MySQL compatibility. We want Dolt to be a drop in replacement for MySQL and this means mimicking MySQL's strengths along with its warts. If we're worried about Regex compatibility off in teh far corners of non-utf8 collations, imagine what [we've already accomplished compatibility-wise](https://docs.dolthub.com/sql-reference/benchmarks/correctness).
```

Comment:
```suggestion
I'm telling you about this because this blog signals how seriously we take MySQL compatibility. We want Dolt to be a drop in replacement for MySQL and this means mimicking MySQL's strengths along with its warts. If we're worried about Regex compatibility off in the far corners of non-utf8 collations, imagine what [we've already accomplished compatibility-wise](https://docs.dolthub.com/sql-reference/benchmarks/correctness).
```